reV/handlers/__init__.py

```diff
@@ -3,7 +3,6 @@
 Sub-package of data handlers
 """
 from .collection import Collector
+from .exclusions import ExclusionLayers
 from .multi_year import MultiYear
-from .resource import Resource
-from .rev_resource import (NSRDB, MultiFileNSRDB, MultiFileWTK,
-                           SolarResource, WindResource)
+from .outputs import Outputs
```

Comment:
Eventually we might want to think about moving collection, multi year, and outputs to rex as well. I'm sure a lot of people in SEAC would start using those tools if they were flexible enough. For now though reV is an easy dependency too.

Reply:
I could see outputs over Collection and MY, but it still feels like more of a reV tool than a rex utility, at least we now have the option for later!